schema/schema.go

```diff
@@ -0,0 +1,20 @@
+package schema
+
+import (
+	"github.com/ONSdigital/dp-kafka/v2/avro"
+)
+
+var reindexRequested = `{
+	"type": "record",
+	"name": "reindex-requested",
+	"fields": [
+		{"name": "job_id", "type": "string", "default": ""},
+    	{"name": "search_index", "type": "string", "default": ""},
```

Comment:
indentation for lines 12 and 13 is a mix of tabs and spaces, so they're not aligned to line 11 which is two tabs